### DISCUSSION TITLE
Arweave

### DISCUSSION THREAD

**Discussion by xloem** [HugPuddle/Apertus#38] [General]
Hey, I wanted to make sure you knew about the arweave chain, which provides for very cheap on-chain storage and offers an http api on every network node.

**Comment by embiimob** [comment]
oh yes, we expect to offer it as an additional storage option, this and IPFS.  #linkallchains

**Comment by embiimob** [comment]
this is probably our last release for the apertus.io 0.3.X code base.   we are developing something new that implments many things we learned while experimenting with it.  

web front end 
distributed lucene based free text searching of blockchain data
remote trusted script deployment with autonomous execution
social media functionality (decentralized twitter, email)
cross chain object trading (NFTs)
polls / voting
state engines
